docs/release-notes/1.5.4/release-1.5.4.md

```diff
@@ -1,4 +1,4 @@
-# [ML.NET](http://dot.net/ml) 1.5.3
+# [ML.NET](http://dot.net/ml) 1.5.4
 
```

Comment:
For the `**AutoML.NET specific fixes**` section:
```md
- Fixed log event handling for AutoML experiments ( [#5554](https://github.com/dotnet/machinelearning/pull/5554) )
```

Granted this bug was both created and fixed since the last version (v1.5.2).